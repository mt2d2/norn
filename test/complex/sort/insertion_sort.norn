def insertion_sort(x: IntAry): IntAry
	return x
end

def main(): Void
	insertion_sort()
end