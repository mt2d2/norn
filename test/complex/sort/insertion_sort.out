couldn't find type by string key: IntAry
